setup.py

```diff
@@ -0,0 +1,20 @@
+import os
+import setuptools
+
+setuptools.setup(
+    name="waving_hands",
+    version="0.1.0",
+    author="Alan Bailey",
+    author_email="",
+    description=(
+        "P2P Multiplayer Python implementation of Richard Bartle's Waving Hands"
+    ),
+    license="GNUv3",
+    keywords="waving hands turnbased",
+    url="https://github.com/alanb33/WavingHands",
+    packages=setuptools.find_packages(),
+    entry_points={
+        "console_scripts": ["waving-hands = waving_hands.waving_hands:main",],
+    },
+    classifiers=[],
+)
```

Comment:
More stuff on packaging here: 
https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/
I think probably other good things to do before putting this on pypi #9 would be:
* Adding classifiers
* double checking the license matches what's expected
* Adding the README to the `long_description` (and maybe converting it to RST)
* Moving version into a version.py file, so waving_hands can reference it some place

But right now it's just nice that it's properly packaging files, which is needed for logging below to work (logging requires a configured module `waving_hands` to control logging levels) and for setting up the `entry_points` above for calling into code using the newly installed `waving-hands` bin script.